com.github.cubitect.cubiomes-viewer.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: com.github.cubitect.cubiomes-viewer
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+command: cubiomes-viewer
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - name: cubiomes-viewer
+    buildsystem: qmake
+    config-opts:
+      - CONFIG+=release
+    post-install:
+      - install -Dm755 cubiomes-viewer /app/bin/cubiomes-viewer
+      - install -Dm644 com.github.cubitect.cubiomes-viewer.metainfo.xml -t /app/share/metainfo/
+      - install -Dm644 etc/com.github.cubitect.cubiomes-viewer.desktop -t /app/share/applications/
+      - install -Dm644 etc/icon.svg /app/share/icons/hicolor/scalable/apps/com.github.cubitect.cubiomes-viewer.svg
+    sources:
+      - type: archive
+        url: https://github.com/Cubitect/cubiomes-viewer/releases/download/2.3.1/cubiomes-viewer-2.3.1-src.tar.gz
+        sha256: e1ba184bec4808b1e25bb416af328440c4f9d78be1c3ac867c14027493cd47e9
+      - type: file
+        path: com.github.cubitect.cubiomes-viewer.metainfo.xml
```

Comment:
while this is the right name, the actual file doesn't have the proper name (it's a typo). So it's not found.